scenarios/happypath/environment/environmentServices.go

```diff
@@ -0,0 +1,25 @@
+package environment
```

Comment:
Change the file name to `environmentservices.go` or `environment-services.go`

Reply:
done